docs/en/getting_started.md

```diff
@@ -14,18 +14,19 @@ Please refer to [installation.md](./installation.md) for installation.
 
 ## Data Preparation
 
-Please refer to [data_preparation.md](./dataset_preparation.md) for data preparation.
+Please refer to [data\_preparation.md](./dataset_preparation.md) for data preparation.
 
 ## Body Model Preparation (Optional)
 
 If you want to obtain keypoints3d, the body model is not necessary.
-If you want to infer SMPL as well, you can prepare the body_model as follows.
+If you want to infer SMPL as well, you can prepare the body\_model as follows.
 
 - [SMPL](https://smpl.is.tue.mpg.de/) v1.0 is used in our experiments.
   - Neutral model can be downloaded from [SMPLify](https://smplify.is.tue.mpg.de/).
   - All body models have to be renamed in `SMPL_{GENDER}.pkl` format. <br/>
     For example, `mv basicModel_neutral_lbs_10_207_0_v1.0.0.pkl SMPL_NEUTRAL.pkl`
-- [smpl_mean_params.npz](https://openmmlab-share.oss-cn-hangzhou.aliyuncs.com/mmhuman3d/models/smpl_mean_params.npz?versionId=CAEQHhiBgICN6M3V6xciIDU1MzUzNjZjZGNiOTQ3OWJiZTJmNThiZmY4NmMxMTM4)
+- [smpl\_mean\_params.npz](https://openmmlab-share.oss-cn-hangzhou.aliyuncs.com/mmhuman3d/models/smpl_mean_params.npz?versionId=CAEQHhiBgICN6M3V6xciIDU1MzUzNjZjZGNiOTQ3OWJiZTJmNThiZmY4NmMxMTM4)
+- [gmm\_08.pkl](https://github.com/vchoutas/smplify-x/files/3295771/gmm_08.zip)
```

Comment:
[Preview link](https://github.com/yl-1993/xrmocap/blob/readthedoc/docs/en/getting_started.md#body-model-preparation-optional). These two lines look the same. Shall we annote their difference? Say `gmm\_08.pkl from smplify-x repo` and `gmm\_08.pkl from openmmlab backup`

Reply:
Good suggestion. After checking their md5, they are exactly the same, i.e., `e3e485788623c5c0e6b7e207c4a55e6f`. Marking them as suggested in this doc.